packages/libs/error-stack/Makefile.toml

```diff
@@ -1,27 +1,78 @@
 extend = { path = "../../../.github/scripts/rust/Makefile.toml" }
```

Comment:
I expect changes to this file to be reverted when `DebugDiagnostics` and the `hooks` feature is removed from `error-stack`